USAGE.md

```diff
@@ -0,0 +1,48 @@
+# config-parse(1) -- screwdriver config parser
+
+## SYNOPSIS
+
+    config-parse [options] <screwdriver.yaml> <job> <task>
+
+## DESCRIPTION
+
+The config-parse utility parses a given `screwdriver.yaml` and writes out the list of steps to run as well as environment variables to set.
+
+It selects the steps and environment variables based on the specified `job` and `task`.  The `job` is the name specified in the `screwdriver.yaml` while the `task` is a SHA1 hash of the JSON representation of the matrix permutation of the job.
+
+For more information on tasks, please see the [domain model](https://screwdriver-cd.github.io/guide/).
+
+## OPTIONS
+
+`-h, --help`
+    output usage information
+
+`-V, --version`
+    output the version number
+
+`-a, --artifactdir <directory>`
+    Specify an alternative artifact directory (default ./artifacts)
+
+## EXAMPLE
+
+To view the output from the main build, run something like the following:
+
+    config-parse ./screwdriver.yaml main a01052005eb33a20120f0118bb46723042218efa
```

Comment:
Oh I see that you have the example here. So why do we have USAGE at 2 different places?